pkgs/development/python-modules/cppyy/default.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  buildPythonPackage,
+  setuptools,
+  wheel,
+  pip,
+  cppyy-cling,
+  cppyy-backend,
+  CPyCppyy,
+}:
+buildPythonPackage rec {
+  pname = "cppyy";
+  version = "3.5.0";
+
+  src = fetchFromGitHub {
+    owner = "wlav";
+    repo = pname;
+    tag = "${pname}-${version}";
+    hash = "sha256-Q40zV7t7iQBJKjqpsvhqcsVtwuYzJkfBRoT1iQxh6rc=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+    pip
+    cppyy-cling
+    cppyy-backend
+    CPyCppyy
+  ];
+
+  propagatedBuildInputs = [
+    cppyy-cling
+    cppyy-backend
+    CPyCppyy
+  ];
+
+  dontUseCmakeConfigure = true;
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
remove with lib; as it increases eval times